$NetBSD: patch-meson.options,v 1.2 2025/03/08 21:26:36 vins Exp $

* Add option to set preferred perl intepreter.
* Allow to pass readline library path. 

--- meson.options.orig	2025-02-08 21:38:17.000000000 +0000
+++ meson.options
@@ -65,6 +65,11 @@ option(
     description: 'Select python implementation',
 )
 option(
+    'perl',
+    type: 'string',
+    description: 'Select perl implementation',
+)
+option(
     'readline',
     type: 'feature',
     value: 'auto',
@@ -106,3 +111,9 @@ option(
     value: 'auto',
     description: 'Enable X PixMap (xpm) support',
 )
+
+option(
+    'readline_lib',
+    type: 'string',
+    description: 'Readline library path',
+)
